### PR TITLE
Fix content__in filter

### PR DESCRIPTION
**Pull request by gerrod3**
fixes: #3634

### CODE REVIEW

File: pulpcore/tests/functional/api/using_plugin/test_repo_versions.py [hunk 1]
```diff
@@ -934,7 +934,7 @@ def test_content_in_repository_version_view(
     pulpcore_bindings,
     file_bindings,
     file_repository_factory,
-    file_random_content_unit,
+    file_content_unit_with_name_factory,
```
Comment (mdellweg) [inline pulpcore/tests/functional/api/using_plugin/test_repo_versions.py:937]: Yes!!!

File: pulpcore/app/viewsets/custom_filters.py [hunk 2]
```diff
@@ -325,6 +325,47 @@ def filter(self, qs, value):
...
+
+        content_units_pks = set(content_units.keys())
+        existing_content_units = Content.objects.filter(pk__in=content_units_pks)
+        raise_for_unknown_content_units(existing_content_units, content_units)
+
+        return qs.with_content(content_units_pks)
+
+
+class WithContentInFilter(BaseInFilter, WithContentFilter):
+    """The multi-content variant of WithContentFilter."""
```
Comment (mdellweg) [inline pulpcore/app/viewsets/custom_filters.py:366]: Is the `BaseInFilter` what fixes the openapi3 schema in this case?

**Review by mdellweg — COMMENTED** [review]
The original bugreport was (only) about the openapi schema being wrong. Have you checked that this fixes it?

File: pulpcore/app/viewsets/custom_filters.py (hunk 2 above)
Comment (gerrod3) [reply pulpcore/app/viewsets/custom_filters.py:366]: Yes, I'll add a schema test for this.

**Review by mdellweg — APPROVED** [review]

**Pull request merged**